packages/fxa-graphql-api/src/gql/account.resolver.ts

```diff
@@ -12,6 +12,7 @@ import {
   ResolveField,
   Resolver,
 } from '@nestjs/graphql';
+import { description } from 'commander';
```

Comment:
Stray import?

Reply:
What the... probably VS Code being too helpful or something